### ISSUE TITLE
AVG thinks that the NVDA update is a threat

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2243]
**Reported by hurrikenny on 2012-04-14 07:17**
In the latest snapshots, when I download the latest copy of NVDA and press OK to install it, it makes a shutting down noise and I think the starting up noise. Next, it looks as though AVG pops up a message that NVDA cannot read, and the following file is what comes up in AVG as a threat: C:\documents and settings\admin\local settings\temp\NSYD.TMP\App\nvda_nouiaccess.exe. This is on version 2012 and running Windows XP home. I borrowed my wife's eyes and she said I need to tab down to Allow and press enter. This has only been happening since the two files have been combined, yet on my Windows 7 64 bit machine with AVG 2012 this message does not come up. Normally I would be able to add this file to my exceptions list, however being a temp file the details keep changing.

**Comment by bhavyashah** [comment]
@feerrenrut @jcsteh Is there anything that needs changing on NVDA's end to prevent AVG (or other anti-virus software) from raising false positives against NVDA? If not, I suggest closing this ticket. @ehollig

**Comment by jcsteh** [comment]
Nothing specific that I know of. This would be a matter of finding a channel to contact relevant anti-virus vendors to report false positives when they arise. Closing.
